.github/workflows/tests.yml

```diff
@@ -405,6 +405,40 @@ jobs:
           name: data-coverage
           path: ./coverage.xml
 
+  lightning-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip && pip install wheel --upgrade
+          pip install -r requirements-ci.txt --upgrade
+          pip install -r requirements-dev.txt --upgrade
+          python -m pip install --index-url https://test.pypi.org/simple/ pennylane-lightning --pre --no-deps
+
+      - name: Install PennyLane
+        run: |
+          python setup.py bdist_wheel
+          pip install dist/PennyLane*.whl

```

Comment:
To avoid the issue of the PennyLane wheel installing an overriding the dev/rc versions on Lightning, it would be safest to allow it to install PennyLane and all dependencies first (lightning included) , and then install the newest Lightning on-top of this without additional deps.

tl;dr Swap the order of these two steps.